src/lib/Header/Header.tsx

```diff
@@ -34,10 +45,23 @@ const useStyles = makeStyles(theme => ({
 }));
 
 
-const Header = ({ logo, contents, page, setPage }) => {
+const Header: React.FC<PropTypes> = ({ logo, contents, page, setPage }) => {
   const classes = useStyles();
 
-  const handleChange = (event, newPage) => {
+  const HeaderTab: React.FC<{ item: string }> = ({ item }) => {
+    const icon = contents[item];
```

Comment:
At this line, `icon` is already `null`, but for some reason it's not accepted in `<Tab />` component.